src/OrchardCore.Modules/OrchardCore.Setup/Services/SetupService.cs

```diff
@@ -74,14 +74,14 @@ IStringLocalizer<SetupService> stringLocalizer
 
         public Task<string> SetupAsync(SetupContext context)
         {
-            var initialState = _shellSettings.State;
+            var initialState = _shellSettings.Status;
```

Comment:
Unrelated, and breaking, should be in another PR.

Reply:
OK

Reply:
Undone via [this commit](https://github.com/OrchardCMS/OrchardCore/pull/1378/commits/40f9fa9798c0e967463773ce645e799fcf7d54e4).